src/components/Login/Logout.jsx

```diff
@@ -27,7 +28,22 @@ const Logout = () => {
         <label id="labelLogout" htmlFor="btnLogout">
           Click the following logout button to log out of your pod:{' '}
         </label>
-        <LogoutButton onLogout={handleLogout} />
+        <button type="submit" onClick={() => setShowConfirmation(true)}>
```

Comment:
On Line 31, change from type="submit" to type="button"

Reply:
Updated to "button" type.